packages/flutter_tools/lib/src/fuchsia/fuchsia_device.dart

```diff
@@ -304,6 +324,12 @@ class FuchsiaDevice extends Device {
     bool ipv6 = false,
     String userIdentifier,
   }) async {
+    if (await isSession) {
+      globals.printTrace('Running on a workstation build.');
```

Comment:
please remove workstation here and replace with something about session_control.